app/page-partials/my-dashboard/users-roles/IdimLookup.tsx

```diff
@@ -1,15 +1,11 @@
-import React, { useState, forwardRef, useEffect, useImperativeHandle } from 'react';
-import { faExclamationCircle, faEye, faDownload, faLock } from '@fortawesome/free-solid-svg-icons';
-import Table from 'components/Table';
+import React, { useState, forwardRef, useEffect } from 'react';
+import { faEye, faDownload } from '@fortawesome/free-solid-svg-icons';
+import Table from 'components/TableNew';
```

Comment:
Is TableNew a sustainable name for this? Or will it eventually replace Table?

Reply:
It will eventually replace the Table as we replace existing Table components in other places